ADOPTERS.md

```diff
@@ -0,0 +1,12 @@
+ To add your organization to this list, simply add your organization's name,
+ optionally with a link. The list is in alphabetical order.
+
+# Organizations Using Helm
+
+- [Blood Orange](https://bloodorange.io)
+- [Microsoft](https://microsoft.com)
+- [IBM](https://www.ibm.com)
+- [Qovery](https://www.qovery.com/)
+- [Samsung SDS](https://www.samsungsds.com/)
+[Ville de Montreal](https://montreal.ca)
```

Comment:
My bad, forgot the intial dash:
```suggestion
- [Ville de Montreal](https://montreal.ca)
```
Signed-off-by: Marc Khouzam <marc.khouzam@montreal.ca>

Reply:
I submitted #7555 for this.